This file is part of MXE. See LICENSE.md for licensing information.

Contains ad hoc patches for cross building.

From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Boris Nagaev <bnagaev@gmail.com>
Date: Sun, 13 Dec 2015 16:56:42 +0300
Subject: [PATCH 1/1] cmake: add headers to install

  * bsdqueue.h
  * miniupnpctypes.h
  * portlistingparse.h

See https://github.com/mxe/mxe/issues/1051#issuecomment-164260560

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1111111..2222222 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -166,6 +166,9 @@ install (FILES
   upnpreplyparse.h
   upnperrors.h
   declspec.h
+  bsdqueue.h
+  miniupnpctypes.h
+  portlistingparse.h
   DESTINATION include/miniupnpc
 )
 
